resolwe_bio/processes/import_data/geneset.yml

```diff
@@ -61,8 +63,10 @@
   name: Create gene set
   requirements:
     expression-engine: jinja
+    resources:
+      network: true
```

Comment:
Check if the network resource is really needed as you are not using re-import in this process.